edge/cmd/edgecore/app/server.go

```diff
@@ -200,7 +202,8 @@ func registerModules(c *v1alpha2.EdgeCoreConfig) {
 	eventbus.Register(c.Modules.EventBus, c.Modules.Edged.HostnameOverride)
 	metamanager.Register(c.Modules.MetaManager)
 	servicebus.Register(c.Modules.ServiceBus)
-	edgestream.Register(c.Modules.EdgeStream, c.Modules.Edged.HostnameOverride, c.Modules.Edged.NodeIP)
+	kubeletHost := net.JoinHostPort(c.Modules.Edged.TailoredKubeletConfig.Address, strconv.Itoa(int(c.Modules.Edged.TailoredKubeletConfig.ReadOnlyPort)))
```

Comment:
probably off topic from this PR, but i would like to ask the question here.
why do we not have `Edged.TailoredKubeletConfig.Port` instead of `Edged.TailoredKubeletConfig.ReadOnlyPort`? `ReadOnlyPort` is the insecured read-only port that kubelet serves on with no authentication/authorization... does that mean `edged` only provides insecured read-only apis? sorry for asking question about implementation of edged...

Reply:
The original design is retained here. My guessing is that because only logs, exec, metrics interfaces are used here, so using ReadOnlyPort, and edged uses the localhost to access the internal kubelet.